sections/api/helpers/is-styled-component.mdx

```diff
@@ -32,7 +32,7 @@ const ParentComponent = styled.div`
   color: royalblue;
 
   ${TargetedComponent} {
-    color: tomato;
```

Comment:
For this one we were using html color names just for legibility, maybe you could find an alternate named color? https://en.wikipedia.org/wiki/Web_colors

Reply:
@probablyup, none of the orange colors do not pass the contrast ratio, how about using any of these colors (and adjusting the examples accordingly)?
![image](https://github.com/styled-components/styled-components-website/assets/12565826/7a356a71-440e-446e-807d-9ed88a662fbe)
Or, I can revert the color changes and merge only the other fixes?